types/builtin-modules/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for builtin-modules 2.0
+// Project: https://github.com/sindresorhus/builtin-modules#readme
+// Definitions by: My Self <https://github.com/me>
```

Comment:
You should write your account here

Reply:
Thanks for the hint, I always forget that...